src/main/resources/application.properties

```diff
@@ -142,3 +142,5 @@ management.health.elasticsearch.enabled=false
 management.health.status.order=DOWN, OUT_OF_SERVICE, UNKNOWN, UP, OUT_OF_SYNC, PROBLEM, COULD_NOT_CONNECT
 
 spring.banner.location=classpath:ccd-data-store-banner.txt
+
+azure.application-insights.instrumentation-key=${APPINSIGHTS_INSTRUMENTATIONKEY:dummy}
```

Comment:
not sure if this is relevant, but checking at the official docs on how to publish swagger it looks like we are a bit out date? 
https://github.com/hmcts/reform-api-docs#publish-swagger-docs
Since we're there shall we follow the latest recommendations?

Reply:
We are using those recommendations only https://github.com/hmcts/ccd-data-store-api/blob/RDM-4064/.travis.yml#L23-L27. 
But the issue is that script try to start app and it is failing atm due to missing appInsights instrumentation property.